.drone/drone.yml

```diff
@@ -507,8 +627,43 @@ volumes:
     path: //./pipe/docker_engine/
   name: docker
 ---
+kind: pipeline
+name: Publish Windows alloy-boringcrypto container
```

Comment:
Should we name the windows ones cngcrypto instead of boring, since that matches our naming?

Reply:
Yeah this is a bug actually, it should've been `alloy-cngcrypto` here. Thanks for catching :)